server/package.json

```diff
@@ -2,9 +2,11 @@
   "name": "polis",
   "version": "0.0.0",
   "description": "polis =====",
-  "main": "app.js",
+  "main": "./dist/app.js",
   "scripts": {
-    "test": "npm format:check",
+    "test": "npm run format:check",
```

Comment:
not sure if the `run` is necessary, but I think it is conventional to include it here.